NachoSpigot-Server/src/main/java/me/rastrian/dev/threads/AsyncThread.java

```diff
@@ -0,0 +1,94 @@
+package me.rastrian.dev.threads;
+
+import io.netty.channel.Channel;
+import io.netty.util.concurrent.Future;
+import io.netty.util.concurrent.GenericFutureListener;
+import me.elier.nachospigot.config.NachoConfig;
+import me.rastrian.dev.threads.netty.Spigot404Write;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import net.minecraft.server.NetworkManager;
+import net.minecraft.server.Packet;
+import net.openhft.affinity.AffinityLock;
+import net.openhft.affinity.AffinityStrategies;
+
+public abstract class AsyncThread {
+    private boolean running = true;
+    private int TICK_TIME = 1000000000 / NachoConfig.combatThreadTPS;
+    private Thread t;
+    protected Queue<Runnable> packets = new ConcurrentLinkedQueue<Runnable>();
+
+    public AsyncThread(String s) {
+        try (final AffinityLock al = AffinityLock.acquireLock();){
+            System.out.println("Lock found.");
```

Comment:
Is this called often? Don't think you'd want this spammed, haha

Reply:
But also, this doesn't flood at all, just run at start.